src/EgressClient.ts

```diff
@@ -59,11 +105,35 @@ export default class EgressClient {
     audioOnly?: boolean,
     videoOnly?: boolean,
     customBaseUrl?: string,
+    disableManifest?: boolean,
```

Comment:
I would remove `disableManifest` from legacy interface, and also mark it as `@deprecated`